public/locales/en/authentication.json

```
@@ -68,7 +68,8 @@
             "submit": "Send the link",
             "errors": {
                 "invalid_email": "This email is invalid"
-            }
+            },
+            "mail_sent": "A link has been sent to your mail address"
```

You use `email` everywhere but here
```suggestion
            "email_sent": "A link has been sent to your email address"
```